### DISCUSSION TITLE
How can i add label, static text or something for static content ? (Base pdf editing not good for me)

### DISCUSSION THREAD

**Discussion by FikretCin** [pdfme/pdfme#120] [General]
Sample:  Name : My Sample Name;

`        "full_name_label": {
                "type": "label",
                "text":"Name :",
                "position": {
                    "x": 1,
                    "y": 1
                },
                "width": 50.04,
                "height": 7,
                "alignment": "left",
                "fontSize": 13,
                "characterSpacing": 0,
                "lineHeight": 1
            },
            "full_name_input": {
                "type": "text",
                "position": {
                    "x":50,
                    "y": 1
                },
                "width": 50.4,
                "height": 7,
                "alignment": "right",
                "fontSize": 13,
                "characterSpacing": 0,
                "lineHeight": 1
            },`

**Comment by hand-dot** [comment]
Hi @FikretCin 
You can use  [  backgroundColor: z.string().optional()](https://github.com/pdfme/pdfme/blob/57886f254b28aae0d26fb5e8a640fd8cdf0df8e8/packages/common/src/schema.ts#L34)  in TextSchema.
